### DISCUSSION TITLE
为什么 HKC SG27Q plus  只能调节对比度,不能调节亮度?

### DISCUSSION THREAD

**Discussion by Yjing367** [xanderfrangos/twinkle-tray#744] [Monitor Compatibility Issues]
我很需要使用Twinkle Tray来调节我的外置显示器亮度,但是我试过了很多版本都不能对我的显示器的亮度实现改变,但是它能准确改变我显示器的亮度的进度条的位置,软件可以实现对比度,红绿蓝的完美控制,就是不能调节亮度,我的显示器菜单里没有找到有关DDC/CI的选项,请问我该怎样才能实现对我的显示器实现亮度控制?谢谢!
![微信图片_20240408223102](https://github.com/xanderfrangos/twinkle-tray/assets/61653176/a7cf31eb-ae34-49ea-93ef-4d70cb564494)
![微信图片_20240408223146](https://github.com/xanderfrangos/twinkle-tray/assets/61653176/1d242b9a-9129-4c46-b108-5fabc6634306)

**Comment by KagaJiankui** [comment]
可以使用[softMCCS](https://entechtaiwan.com/lib/softmccs.shtm)查看这个显示器支持的ddc/ci指令, 调节无效有可能是因为该显示器控制亮度的ddc/ci和常见指令不同

**Comment by DogHeadOrdinaryPassBy** [comment]
同款显示器同样的问题,等一个后续

**Reply by CRHcs** [reply]
开了hdr就只能调音量，关了就可以调亮度了

**Reply by DogHeadOrdinaryPassBy** [reply]
谢谢你的回答,不过我尝试了更新twinkle tray至最新版并且关闭hdr后,再次操作仍然没反应,可能是因为型号不同吧

**Comment by CRHcs** [comment]
开了hdr就只能调音量，关了就可以调亮度了

**Comment by CRHcs** [comment]
我的型号是hkc gs27qk，我也更新到最新了，还是一样

________________________________
发件人: DogHeadOrdinaryPassBy ***@***.***>
发送时间: 2024年12月9日 11:08
收件人: xanderfrangos/twinkle-tray ***@***.***>
抄送: cs ***@***.***>; Comment ***@***.***>
主题: Re: [xanderfrangos/twinkle-tray] 为什么 HKC SG27Q plus 只能调节对比度,不能调节亮度? (Discussion #744)


谢谢你的回答,不过我尝试了更新twinkle tray至最新版并且关闭hdr后,再次操作仍然没反应,可能是因为型号不同吧

―
